src/tasks/daily.ts

```diff
@@ -643,6 +669,11 @@ const DailyTasks: GarboTask[] = [
     completed: () => snojoConfigured,
     do: configureSnojo,
   },
+  {
+    name: "Determine whether to use free banishers for embezzlers",
+    completed: () => freeBanishesChecked,
+    do: determineFreeBunnyBanish,
```

Comment:
Does this need to be a grimoire task? It doesn't actually make any network queries as far as I can tell nor does it read or change the state of the game, only the state of kolmafia.

Reply:
So I'm not sure how viable it is, but I would basically have it get calculated the first time we ask about this information, instead of calculating it separately. Like a getter